plugins/modules/edpm_container_manage.py

```diff
@@ -288,7 +298,6 @@ def manage_container(self, name, config):
             'debug': self.debug,
             'log_driver': 'journald',
             'log_level': 'info',
-            'log_opt': {"path": f"{self.log_base_path}/{name}.log"},
```

Comment:
by dropping this, the functionality is removed, not just deprecated. Could we leave this in place until the next release?

Reply:
Not really. `log_opt/path` is not relevant when using journald log driver[1]. As I mentioned in the earlier comment, the functionality has been removed since we switched the `log_driver`.

[1] https://github.com/containers/podman/blob/main/pkg/specgen/specgen.go#L22-L25

Reply:
No, this lgtm, thanks.